monkey_island/cc/services/config.py

```diff
@@ -440,10 +440,16 @@
                             "default": "/tmp/monkey",
                             "description": "Determines where should the dropper place the monkey on a Linux machine"
                         },
-                        "dropper_target_path": {
-                            "title": "Dropper target path on Windows",
+                        "dropper_target_path_win_32": {
+                            "title": "Dropper target path on Windows (32bit)",
                             "type": "string",
-                            "default": "C:\\Windows\\monkey.exe",
+                            "default": "C:\\Windows\\monkey32.exe",
+                            "description": "Determines where should the dropper place the monkey on a Windows machine"
```

Comment:
Maybe also add a note on the 32/64 in the description?

Reply:
fixed